src/renderer/MeshRenderer.js

```diff
@@ -12,13 +12,17 @@ export default class MeshRenderer extends BaseRenderer {
     super(type);
 
     this.container = container;
-
     this._targetPool = new Pool();
     this._materialPool = new Pool();
     this._body = new THREE.Mesh(
       new THREE.BoxGeometry(50, 50, 50),
       new THREE.MeshLambertMaterial({ color: '#ff0000' })
     );
+    this.ThreeSprite = THREE.Sprite;
+  }
+
+  isThreeSprite(particle) {
+    return particle.target instanceof this.ThreeSprite;
```

Comment:
You might wanna use the [isSprite](https://github.com/mrdoob/three.js/blob/master/src/objects/Sprite.js#L70), as instanceof is a lot slower than property checks.

Reply:
Done, thanks!